### ISSUE TITLE
Previous/Next buttons are not disabled when at beginning/end of epub

### ISSUE THREAD

**Issue by danielweck** [readium/readium-js-viewer#72]
The previous and next buttons should be disabled when they won't do anything such as at the beginning and end of the epub. This can be especially confusing for people using a screen reader as they will hear the button title but activating it will not do anything.

Bugzilla:
https://app.devzing.com/Readium/bugzilla/show_bug.cgi?id=79
